### ISSUE TITLE
Compatibility with `mtl ^>= 2.3.1`

### ISSUE THREAD

**Issue by sjakobi** [snapframework/snap#223]
https://hackage.haskell.org/package/mtl-2.3.1/changelog

This will apparently require code changes:

```
src/Snap/Snaplet/Internal/Types.hs:24:62: error:
    Module ‘Control.Monad.Reader’ does not export ‘MonadPlus’
   |
24 | import           Control.Monad.Reader         (MonadIO (..), MonadPlus, MonadReader (ask, local))
   |                                                              ^^^^^^^^^
```

**Comment by andreasabel** [comment]
Me too.

**Comment by andreasabel** [comment]
This issue is fixed in 1.1.3.2 and can be closed.
